probe-rs-tools/src/bin/probe-rs/cmd/gdb_server/mod.rs

```diff
@@ -83,7 +84,27 @@ impl Cmd {
             }
             cmd.args(self.gdb_args);
             eprintln!("Spawning {cmd:?}");
-            Some(cmd.spawn()?)
+            let gdb = Arc::new(Mutex::new(cmd.spawn()?));
+
+            let gdb2 = gdb.clone();
+            tokio::spawn(async move {
+                let mut last_ctrl_c = Instant::now() - Duration::new(100, 0);
+                loop {
+                    // Don't exit on ctrl-c as you need to use this key combination
+                    // to ask gdb to interrupt execution of the tracee.
+                    tokio::signal::ctrl_c().await.unwrap();
+                    if last_ctrl_c.elapsed() < Duration::new(0, 500_000_000) {
```

Comment:
Please use some method that mentions a physical unit, like `Duration::from_millis`